classes/services/PKPContextService.inc.php

```diff
@@ -299,6 +299,43 @@ public function validate($action, $props, $allowedLocales, $primaryLocale) {
 			}
 		});
 
+		// Ensure that the form and submission locales are available for the context
+		$validator->after(function($validator) use ($action, $props) {
+			$context = Application::get()->getRequest()->getContext();
+			if (isset($props['id']) && $context->getId() !== $props['id']) {
+				$context = $this->get($props['id']);
+			}
+
+			$getCurrentPropValue = function($prop) use ($props, $context) {
+				if (isset($props[$prop])) {
```

Comment:
The `??` operator would make this function a lot more concise:
```
$props[$prop] ?? $context->getData($prop)
```

Reply:
Indeed, but I ain't the biggest fan of the [null coalescing operator](https://www.php.net/manual/en/migration70.new-features.php#migration70.new-features.null-coalesce-op). So I don't use it a lot, always opting for `if` usage instead; letting the code more "readable" and verbose.

Reply:
We have room for that kind of stylistic preference in such a large codebase :)